<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Choose your side</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="choose_label">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>10</y>
      <width>230</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose your side!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>460</y>
      <width>130</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rebellion_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>340</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="empire_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>340</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
